specification.markdown

```diff
@@ -404,23 +404,36 @@ PathMaxOccurrence     = 1*DIGIT ; no upper limit
 
 The `*`, `(` and `)` symbols are interpreted as defined in [@!RFC5234].
 
-The EBMLPathAtom part of the EBMLElementPath MUST be equal to the name attribute of the EBML Schema.
+The EBMLAtomName part of the EBMLLastPath MUST be equal to the `@name` attribute of the EBML Schema.
+If the EBMLLastPath is an EBMLPathAtomRecursive, the EBML Element can occur within itself recursively (see (#recursive)).
 
 The starting PathDelimiter of the path corresponds to the root of the EBML Document.
 
-In some cases the EBMLLastParent part of the path is an EBMLGlobalParent. A path with a EBMLGlobalParent defines a Global Element; see (#global-elements). Any path that starts with the EBMLFixedParent of the Global Element and matches the occurrences found in the GlobalParentOccurence is a valid path for the Global Element. 
+The `@path` value MUST be unique within the EBML Schema. The `@id` value corresponding to this `@path` MUST NOT be defined for use within another EBML Element with the same EBMLParentPath as this `@path`.
 
-The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition. The repetition amounts correspond to the amount of unspecified Parent Element levels there can be between the EBMLFixedParent and the actual EBMLElementPath.
+In some cases the path contains a GlobalPlaceholder.
+A path with a GlobalPlaceholder defines a Global Element; see (#global-elements).
+If GlobalPlaceholder is not present then the Element is not a Global Element and only has one fixed EBMLParentPath.
+Any path that starts with the EBMLParentPath of the Global Element and matches the occurrences found in the GlobalParentOccurence is a valid path for the Global Element.
 
-PathMinOccurrence represents the minimum number of element path required between the EBMLFixedParent and the Global Element EBMLElementPath. For example 0 means the EBMLElementPath can be right after the EBMLFixedParent, 1 means there has to be at least an element between the EBMLFixedParent and the EBMLElementPath. If PathMinOccurrence is not present then that EBML Element has an PathMinOccurrence value of 0.
+The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition.
+The repetition amounts correspond to the amount of unspecified valid Elements that can be found in place of the GlobalPlaceholder for the path to be valid.
 
-PathMaxOccurrence represents the maximum number of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath. It cannot have the value 0 as it would be the Global Element can only be found right after the EBMLFixedParent, in which case it's not a Global Element anymore. If PathMaxOccurrence is not present then there is no upper bound for the permitted number of occurrences of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath.
+PathMinOccurrence represents the minimum number of path elements required between the EBMLParentPath and the EBMLLastPath.
```

Comment:
But what does this mean if there is no EBMLParentPath (it is optional)?

Reply:
Yes it's optional. And in the only cases it's used CRC-32 and Void it is empty. In that case it's the root.
Maybe we can mention that an empty `EBMLParentPath` means it's the root.

Alternatively we can rework the ABNF so that the root is always present. In that case the EBMLLastPath would not contain the `\` anymore.

Reply:
Hi @robUx4, I tried with the latter option and making EBMLParentPath mandatory, where that can be either `\` as Root or the series of intermediate paths. Please review.